docs/dsp-tools-create.md

```diff
@@ -250,194 +251,201 @@ A resource can be assigned to a taxonomic node within its properties. So a resou
 title "La Traviata" would have the property/attribute "musical-genre" with the value "Grand opera". Within the DSP,
 each property or attribute has an assigned cardinality. Sometimes, a taxonomy allows that an object may belong to 
 different categories at the same time (e.g. an image which depicts several categories at the same time). In these cases,
-a cardinality &gt; 1 allows to add multiple attributes
-of the same time. See further below the description of the [cardinalities](#cardinalities)
+a cardinality &gt; 1 allows to add multiple attributes of the same time. See further below the description of the
```

Comment:
do characters like `<` need to be escaped as html entities in markdown?  
In any case, I'd either have it in back ticks (`cardinality > 1`) or write it out ("cardinality greater than one")